components/clp-package-utils/clp_package_utils/scripts/native/search.py

```diff
@@ -9,6 +9,7 @@
 import sys
```

Comment:
Missing standard library imports.
```suggestion
import ipaddress
import socket
import sys
```

Reply:
they have been imported at Line 5 & Line 8 (the original implementation also uses those libs